src/components/dashboard/Reason.js

```diff
@@ -74,12 +74,13 @@ const SendReason = (props: AmountProps) => {
   const next = useCallback(() => {
     const [nextRoute, ...nextRoutes] = screenState.nextRoutes || []
 
-    props.screenProps.push(nextRoute, {
-      nextRoutes,
-      ...restState,
-      reason,
-      params,
-    })
+    screenState.category &&
```

Comment:
shouldnt next button be simply disabled?

Reply:
It's disabled, but the input has the **next** function as **onSubmitEditing** method.

Reply:
ohh so i think user should be able to continue only via pressing next...
what do you  think? @julianpolcode @serdiukov-o-nordwhale

Reply:
I agree